applications/openshift/etcd/etcd_check_cipher_suite/tests/etcd_cipher_suite.fail.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# remediation = none
+
+yum install -y jq
+
+kube_apipath="/kubernetes-api-resources"
+
+
+# Create infra file for CPE to pass
+mkdir -p "$kube_apipath/api/v1/namespaces/openshift-etcd/configmaps"
+etcd_config_apipath="$kube_apipath/api/v1/namespaces/openshift-etcd/configmaps/etcd-pod"
```

Comment:
Hi @lmzuccarelli 
I have a doubt if we need to add $kube_apipath in the path here.
Ultimately in filepath, L35, we would get `/kubernetes-api-resources/kubernetes-api-resources/api/v1/namespaces/openshift-etcd/configmaps/etcd-pod` (twice kubernetes-api-resources), which might mess up the filename expected by the scanner.
I'm pretty puzzled because the test passes in spite of the duplication. Actually both work!